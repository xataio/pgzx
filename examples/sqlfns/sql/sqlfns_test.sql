CREATE EXTENSION sqlfns;

SET search_path TO sqlfns;

SELECT hello_world_c(NULL);
SELECT hello_world_c('pgzx');

SELECT hello_world_zig(NULL);
SELECT hello_world_zig('pgzx');

SELECT hello_world_zig(NULL);
SELECT hello_world_zig('pgzx');

SELECT hello_world_zig_null(NULL);
SELECT hello_world_zig_null('pgzx');

SELECT hello_world_zig_datum(NULL);
SELECT hello_world_zig_datum('pgzx');

SELECT hello_world_anon(NULL);
SELECT hello_world_anon('pgzx');

SELECT hello_world_mod(NULL);
SELECT hello_world_mod('pgzx');

SELECT hello_world_file(NULL);
SELECT hello_world_file('pgzx');

DROP EXTENSION sqlfns;
