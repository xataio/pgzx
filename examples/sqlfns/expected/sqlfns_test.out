CREATE EXTENSION sqlfns;
SET search_path TO sqlfns;
SELECT hello_world_c(NULL);
 hello_world_c 
---------------
 Hello World
(1 row)

SELECT hello_world_c('pgzx');
 hello_world_c 
---------------
 Hello, pgzx!
(1 row)

SELECT hello_world_zig(NULL);
 hello_world_zig 
-----------------
 Hello World
(1 row)

SELECT hello_world_zig('pgzx');
 hello_world_zig 
-----------------
 Hello, pgzx!
(1 row)

SELECT hello_world_zig(NULL);
 hello_world_zig 
-----------------
 Hello World
(1 row)

SELECT hello_world_zig('pgzx');
 hello_world_zig 
-----------------
 Hello, pgzx!
(1 row)

SELECT hello_world_zig_null(NULL);
 hello_world_zig_null 
----------------------
 
(1 row)

SELECT hello_world_zig_null('pgzx');
 hello_world_zig_null 
----------------------
 Hello, pgzx!
(1 row)

SELECT hello_world_zig_datum(NULL);
 hello_world_zig_datum 
-----------------------
 
(1 row)

SELECT hello_world_zig_datum('pgzx');
 hello_world_zig_datum 
-----------------------
 Hello, pgzx!
(1 row)

SELECT hello_world_anon(NULL);
 hello_world_anon 
------------------
 Hello World
(1 row)

SELECT hello_world_anon('pgzx');
 hello_world_anon 
------------------
 Hello, pgzx!
(1 row)

SELECT hello_world_mod(NULL);
 hello_world_mod 
-----------------
 Hello World
(1 row)

SELECT hello_world_mod('pgzx');
 hello_world_mod 
-----------------
 Hello, pgzx!
(1 row)

SELECT hello_world_file(NULL);
 hello_world_file 
------------------
 Hello World
(1 row)

SELECT hello_world_file('pgzx');
 hello_world_file 
------------------
 Hello, pgzx!
(1 row)

DROP EXTENSION sqlfns;
